components/LeagueSurvivors/LeagueSurvivors.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { cn } from '@/utils/utils';
+import { ILeagueSurvivorsProps } from './LeagueSurvivors.interface';
+import * as React from 'react';
+
+const LeagueSurvivors = React.forwardRef<
+  HTMLParagraphElement,
+  ILeagueSurvivorsProps
+>(({ className, survivors, totalPlayers, ...props }) => (
```

Comment:
Why are we allowing `className` as a prop? This component should be set styles without the ability to change the styles.

Reply:
Because `LeagueSurvivors` is used in different places with different font sizes, weights, etc so it is not a "one size fits all" component.

Reply:
Correct. Do not allow manual classNames. UI libraries should control the styles of their framework.